src/hooks/useAvatarUpload.spec.ts

```
@@ -271,8 +271,9 @@ describe('useAvatarUpload', () => {
 
       // Re-import the hook with mocked dependency
       vi.resetModules();
-      const { useAvatarUpload: useAvatarUploadMocked } =
-        await import('./useAvatarUpload');
+      const { useAvatarUpload: useAvatarUploadMocked } = await import(
+        './useAvatarUpload'
+      );
```

ESLint

[error] Replace `·await·import(⏎········'./useAvatarUpload'⏎······` with `⏎········await·import('./useAvatarUpload'`

(prettier/prettier)